src/libs/actions/Wallet.js

```diff
@@ -173,8 +173,8 @@ function updatePersonalDetails(personalDetails) {
     const ssn = personalDetails.ssn || '';
     const phoneNumber = personalDetails.phoneNumber || '';
     API.write('UpdatePersonalDetailsForWallet', {
-        firstName,
-        lastName,
+        legalFirstName: firstName,
+        legalLastName: lastName,
```

Comment:
This is being handled here: https://github.com/Expensify/Web-Expensify/pull/34655/files#diff-8367f109728fcc54db479ccc765a52d2115e52d35eba9c660e665be1a34411acR2737

Reply:
Thanks!